pom.xml

```diff
@@ -5,7 +5,7 @@
     <groupId>org.sagebionetworks.bridge</groupId>
     <artifactId>sdk-group</artifactId>
     <!-- Please use the maven versions plugin to manage this, e.g.`mvn versions:set -DnewVersion=0.8.1`-->
-    <version>0.12.47</version>
+    <version>0.12.48</version>
```

Comment:
It looks like another change has already bumped the version to 12.48, and another change will bump this to 12.49. We'll need to bump this to 12.50, and we'll need to coordinate between our pull requests to make sure we don't stomp on each other.